src/libraries/System.Runtime.InteropServices.JavaScript/src/System/Runtime/InteropServices/JavaScript/Marshaling/JSMarshalerArgument.BigInt64.cs

```diff
@@ -11,6 +11,7 @@ public partial struct JSMarshalerArgument
         /// Implementation of the argument marshaling.
         /// It's used by JSImport code generator and should not be used by developers in source code.
         /// </summary>
+        /// <param name="value">Value to be marshaled.</param>
```

Comment:
Suggestion applies to all similar texts:
```suggestion
        /// <param name="value">The value to be marshaled.</param>
```